duckduckgo/iOS
# Re-Enable Broken Site Toast on iOS

Task/Issue URL: https://app.asana.com/0/1163321984198618/1207889813347127/f
Tech Design URL:
CC:

**Description**:



**Steps to test this PR**:
1. Got to a page
2. Refresh twice quickly,  you should see the toast prompt.
3. If you do it again it should not show (7 day cool down)
4. By resetting the last shown date user default you can test that dismissing 3 times will stop showing the prompt for 30 days.



**Definition of Done (Internal Only)**:


**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
